PACKAGES.md

```diff
@@ -44,19 +64,22 @@ They must be published in the traditional way, as a bundle.
 This bundle must work on both CommonJS and ESM environments.
 
 
-## Feature packages
+## Add-on packages
 
 -   `@plone/blocks`
--   `@plone/contents`
 -   `@plone/slots`
+-   `@plone/theming`
+-   `@plone/contents`
```

Comment:
Can you leave these sorted alphabetically?

Reply:
Same as above, but if you think it's valuable, let's do it.

Reply:
Yeah, relevance is arbitrary to the developer. Alpha also makes it easier to find in a list.